src/solvers/umfpack.jl

```diff
@@ -209,39 +209,45 @@ workspace_W_size(F::UmfpackLU) = workspace_W_size(F, has_refinement(F))
 workspace_W_size(S::Union{UmfpackLU{<:AbstractFloat}, SparseMatrixCSC{<:AbstractFloat}}, refinement::Bool) = refinement ? 5 * size(S, 2) : size(S, 2)
 workspace_W_size(S::Union{UmfpackLU{<:Complex}, SparseMatrixCSC{<:Complex}}, refinement::Bool) = refinement ? 10 * size(S, 2) : 4 * size(S, 2)
 
+const ATLU{Tv, Ti} = Union{Transpose{Tv, UmfpackLU{Tv, Ti}}, Adjoint{Tv, UmfpackLU{Tv, Ti}}}
```

Comment:
Seems like you don't need the type parameters below, so

```suggestion
const ATLU = Union{Transpose{<:Any, UmfpackLU}, Adjoint{<:Any, UmfpackLU}}
```

might be enough? Not sure about `<:Any` vs. `Any` above.

Reply:
seems resonable